go/store/prolly/message/prolly_map.go

```diff
@@ -63,6 +65,10 @@ func (s ProllyMapSerializer) Serialize(keys, values [][]byte, subtrees []uint64,
 		valTups = writeItemBytes(b, values, valSz)
 		serial.ProllyTreeNodeStartValueOffsetsVector(b, len(values)-1)
 		valOffs = writeItemOffsets(b, values, valSz)
+		if len(s.ValDesc.Addrs) > 0 {
+			serial.ProllyTreeNodeStartValueOffsetsVector(b, len(values)*len(s.ValDesc.Addrs))
```

Comment:
we're using `TupleDesc.Addrs` to measure the size of `value_address_offsets`, but it looks like we never populate `TupleDesc.Addrs`.

Also, you want to use `ProllyTreeNodeStartValueAddressOffsetsVector` instead.